bin/run_hvac.py

```diff
@@ -1,9 +1,9 @@
 #!/usr/bin/env python
 # This file is part of ts_hvac.
 #
-# Developed for the LSST Telescope and Site Systems.
-# This product includes software developed by the LSST Project
-# (https://www.lsst.org).
+# Developed for the Vera C. Rubin Observatory Telescope and Site Systems.
```

Comment:
Minor picky point: I thought we were using `Vera Rubin Observatory`? It's no big deal, but it'd be nice if we all did the same thing for the boilerplate.

Reply:
Good point. Perhaps we should raise this in parking lot in a standup next week when most people are back from their vacations.